### PR TITLE
google sign in

### PR DESCRIPTION
**Pull request by adsau59**
related issue: #23 

Creates way to sign into the app using google signin method

### CODE REVIEW

File: code/presentation/mb-client/src/core/auth/googleSIgnIn.ts
Comment (electr0de) [inline code/presentation/mb-client/src/core/auth/googleSIgnIn.ts:1]: The file name convention is kebab case so change it to google-sign-in.ts

**Review by electr0de — COMMENTED** [review]
Other than that looks fine to me

File: code/presentation/mb-client/src/core/auth/googleSIgnIn.ts
Comment (adsau59) [reply code/presentation/mb-client/src/core/auth/googleSIgnIn.ts:1]: its a library, how can i change it?

Comment (adsau59) [reply code/presentation/mb-client/src/core/auth/googleSIgnIn.ts:1]: done

**Pull request merged**